src/update-0.8.md

```diff
@@ -49,44 +57,60 @@ let a: u32 = Standard.sample(&mut rng);
 let b: u32 = Standard.sample(&mut rng);
 ```
 
-[`Rng::gen_range`] now takes a `Range` instead of two numbers. This requires
-replacing `gen_range(a, b)` with `gen_range(a..b)` in code written for `rand
-0.7`. We suggest to replace the regular expression
-`gen_range\(([^,]*),\s*([^)]*)\)` with `gen_range(\1..\2)` (or
-`gen_range($1..$2)` in some tools without support for backreferences).
-Additionally, if `a` or `b` were a reference, explicit dereferencing may now be
-required. Inclusive ranges are now supported: `gen_range(a, b + 1)` can be
-replaced with `gen_range(a..=b)`.
+#### gen_range
```

Comment:
Maybe `gen_range` is better?

Reply:
I didn't realise in-line code was supported in titles (don't think it is on GH). Works here though.